downstream/titles/eda/eda-user-guide/master.adoc

```diff
@@ -8,12 +8,9 @@ include::attributes/attributes.adoc[]
 // Book Title
 = {EDAName} Controller user guide
```

Comment:
Suggest not using the attribute in the master.adoc book title because it can sometimes cause issues with the Pantheon build.